src/GraphQL/Queries/GetListingsQuery.php

```diff
@@ -72,6 +72,15 @@ public function args(): array
                 'type' => GraphQL::type('MultiTokenIdInput'),
                 'description' => __('enjin-platform-marketplace::type.marketplace_listing.field.takeAssetId'),
             ],
+            'collectionId' => [
+                'type' => GraphQL::type('BigInt'),
+                'description' => __('enjin-platform::input_type.multi_token_id.field.collectionId'),
+                'rules' => [new MinBigInt(), new MaxBigInt(Hex::MAX_UINT128)],
+            ],
```

Comment:
I know it was me that said `collectionId` but now that I looked your PR and the whole code. I wonder if it should be an array? I mean we have: `listingIds`, `states`, `ids`. Maybe an array of `collectionIds` would be better? So the person doesn't need to query more than one time (let's say it wants to query its own collections). Unless you think that would reduce the performance too much.